src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+p,
+h2,
+body,
+main,
+ul {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
it is better to move these lines to a separate file - reset.scss

Reply:
fixed